app/Platform/Controllers/DeveloperGameListController.php

```diff
@@ -0,0 +1,191 @@
+
+            $game['NumAuthoredLeaderboards'] = $gameAuthoredLeaderboardsList[$gameModel->ID] ?? 0;
+            $game['leaderboards_count'] = $gameModel->leaderboards_count;
+
+            $gameTickets = $gameTicketsList[$gameModel->ID] ?? null;
+            $game['NumTickets'] = $gameTickets['NumTickets'] ?? 0;
+            $game['NumAuthoredTickets'] = $gameTickets['NumAuthoredTickets'] ?? 0;
+
+            $gameProgress = $userProgress[$gameModel->ID]['achievements_unlocked_hardcore'] ?? 0;
+            $game['CompletionPercentage'] = $gameProgress * 100 / $gameModel->achievements_published;
+
+            $game['RetroRatio'] = $gameModel->TotalTruePoints / $gameModel->points_total;
+
+            $game['ConsoleName'] = $consoles->firstWhere('ID', $game['ConsoleID'])->Name;
+            $game['SortTitle'] = $game['Title'];
+            if (substr($game['Title'], 0, 1) == '~') {
+                $tilde = strrpos($game['Title'], '~');
+                $game['SortTitle'] = trim(substr($game['Title'], $tilde + 1) . ' ' . substr($game['Title'], 0, $tilde + 1));
+            }
+            $games[] = $game;
+        }
+
+        if ($filterOptions['sole']) {
+            $games = array_filter($games, function ($game) {
+                return $game['NumAuthoredAchievements'] == $game['achievements_published']
+                        && $game['NumAuthoredLeaderboards'] == $game['leaderboards_count'];
+            });
+        }
+
+        $reverse = substr($sortOrder, 0, 1) === '-';
+        $sortMatch = $reverse ? substr($sortOrder, 1) : $sortOrder;
+        $sortFunction = match ($sortMatch) {
+            default => function ($a, $b) {
+                return $a['SortTitle'] <=> $b['SortTitle'];
+            },
+            'achievements' => function ($a, $b) {
+                return $a['NumAuthoredAchievements'] <=> $b['NumAuthoredAchievements'];
+            },
+            'points' => function ($a, $b) {
+                return $a['NumAuthoredPoints'] <=> $b['NumAuthoredPoints'];
+            },
+            'retroratio' => function ($a, $b) {
+                return $a['RetroRatio'] <=> $b['RetroRatio'];
+            },
+            'leaderboards' => function ($a, $b) {
+                return $a['NumAuthoredLeaderboards'] <=> $b['NumAuthoredLeaderboards'];
+            },
+            'players' => function ($a, $b) {
+                return $a['players_total'] <=> $b['players_total'];
+            },
+            'tickets' => function ($a, $b) {
+                return $a['NumAuthoredTickets'] <=> $b['NumTickets'];
+            },
+            'progress' => function ($a, $b) {
+                return $a['CompletionPercentage'] <=> $b['CompletionPercentage'];
+            },
+        };
+        usort($games, $sortFunction);
+        if ($reverse) {
+            $games = array_reverse($games);
+        }
+
+        return view('community.components.developer.sets-page', [
```

Comment:
The controller is in the Platform domain, seems like this should be:

`platform.components.developer.sets-page`

Reply:
I really don't understand the domains. Users are in Community, and there's already a developers component subdirectory there. And there's already a DeveloperController in the Platform domain, though it doesn't seem to be tied to anything.

Reply:
Community vs Platform domains are meant to distinguish between core platform features and community "addons". Developers are a weird thing in-between, i think we can move most of it to platform though?

Minor: `DeveloperGamesController` - `List` can be omitted imo